src/Okta.Sdk.UnitTests/DefaultRequestExecutorShould.cs

```diff
@@ -85,5 +85,75 @@ public async Task NotRetryRequestOnceWhenResponseIsNot401AndAuthorizationModeIsP
             tokenMessageHandler.NumberOfCalls.Should().Be(1);
             requestMessageHandler.NumberOfCalls.Should().Be(1);
         }
+
+        [Theory]
+        [InlineData(HttpVerb.Get)]
+        [InlineData(HttpVerb.Post)]
+        [InlineData(HttpVerb.Put)]
+        [InlineData(HttpVerb.Delete)]
+        public async Task ExecuteMethodBasedOnVerb(HttpVerb verb)
+        {
+            var requestMessageHandler = new MockHttpMessageHandler(string.Empty, HttpStatusCode.OK);
```

Comment:
I think we can simplify this test by removing OAuth related code unless you specifically want to test expected verbs when using OAuth.

Reply:
I'll see about removing those bits.  I took a lot of the setup code from other tests and wasn't sure which parts were specifically needed for the mocks to function correctly.